.mergify.yml

```diff
@@ -1,7 +1,7 @@
 pull_request_rules:
   - name: automatic merge
     conditions:
-      - "status-success=buildkite/rules-sh"
+      - "check-success=Build & Test"
```

Comment:
Does this cover all 3 configurations?
I can see that in `rules_haskell` mergify rules specify checks for all jobs generated by `matrix` strategy.
https://github.com/tweag/rules_haskell/blob/688e330cdc50161d916d2259d1fe979ae820b3cc/.mergify.yml#L4

Reply:
Thank you, I got that mixed up. I've fixed the conditions. I also saw in the Mergify docs that [strict mode](https://blog.mergify.com/strict-mode-deprecation/) will be removed soon, so, I've used this opportunity to update to the new ways.
I've checked the new rules on this PR in the [Mergify Dashboard](https://dashboard.mergify.com/github/tweag/repo/rules_sh/config-editor?repositoryName=rules_sh&pullRequestNumber=17).